### PR TITLE
task 11 complete

### PR DESCRIPTION
**Pull request by kajihan**

### CODE REVIEW

File: src/main/java/pro/java/hw11/Main.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        int count = 0;
+        for (String str : stringList) {
+            if (str.equals(targetString)) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public static <T> List<T> ArrayToListConverter(T[] array) {
```
Comment (kartius) [inline src/main/java/pro/java/hw11/Main.java]: Please, don't use capital symbol in the name of methods

File: src/main/java/pro/java/hw11/Main.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
+package pro.java.hw11;
+
+import java.util.*;
+
+import static pro.java.hw11.WordOccurrence.*;
```
Comment (kartius) [inline src/main/java/pro/java/hw11/Main.java]: Please, use single import instead of *

File: src/main/java/pro/java/hw11/WordOccurrence.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
+package pro.java.hw11;
+
+import java.util.*;
```
Comment (kartius) [inline src/main/java/pro/java/hw11/WordOccurrence.java:3]: The same here

File: src/main/java/pro/java/hw11/WordOccurrence.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        }
+        // Sort results by name
+        Collections.sort(wordOccurrences);
+        return wordOccurrences;
+    }
+
+    public static void printOccurrence(List<WordOccurrence> result) {
+        // Displaying the result with commas after each occurrence and nothing after the last
+        for (WordOccurrence occurrence : result) {
+            if (occurrence == result.get(result.size() - 1)) {
```
Comment (kartius) [inline src/main/java/pro/java/hw11/WordOccurrence.java]: equals instead of == ?

File: src/main/java/pro/java/hw11/Main.java (hunk 1 above)
Comment (kajihan) [reply src/main/java/pro/java/hw11/Main.java]: fixed

File: src/main/java/pro/java/hw11/Main.java (hunk 2 above)
Comment (kajihan) [reply src/main/java/pro/java/hw11/Main.java]: fixed

File: src/main/java/pro/java/hw11/WordOccurrence.java (hunk 3 above)
Comment (kajihan) [reply src/main/java/pro/java/hw11/WordOccurrence.java:3]: fixed

File: src/main/java/pro/java/hw11/WordOccurrence.java (hunk 4 above)
Comment (kajihan) [reply src/main/java/pro/java/hw11/WordOccurrence.java]: fixed

**Review by kartius — APPROVED** [review]

**Pull request merged**